{{cookiecutter.repostory_name}}/setup-dev.sh

```diff
@@ -1,25 +1,39 @@
 #!/usr/bin/env bash
 # Copyright 2017, Reef Technologies (reef.pl), All rights reserved.
 
+set -euo pipefail
+
 PROJECT_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 ENV_DIR="./envs/dev"
 # shellcheck disable=SC2164
 cd "${PROJECT_DIR}"
 
-# Create a lock file, install Python dependencies
-[ -f pdm.lock ] || pdm lock --group :all
+# Workaround for PDM which sometimes creates a 3.10 venv
+if [[ ! -d ".venv" ]]; then
+    python3.11 -m venv .venv
+fi
```

Comment:
please put pdm issue link here so we can know when we can remove this workaround

Reply:
I searched and I could find any relevant issue. Created one now: https://github.com/pdm-project/pdm/issues/2789

Reply:
Please put the link in workaround comment

Reply:
done